Transfer to Pots

rossmuego opened the pull request:
Ability to transfer money into a pot from the menu. All you need to do is click on the pot, then select an option for how much to transfer. 

![image](https://user-images.githubusercontent.com/17198112/51151121-652f3e00-1861-11e9-98e9-b41e3e8d4574.png)

johneas10 commented:
Good work mate! Pretty busy at the moment and then I'm on holiday from Friday, but I'll try and review properly asap.

But just a few things after a quick look:

- can you add `.DS_Store` to `.gitignore` so those files don't get committed?
- although it looks like you've updated the PR to remove the files, we should probably add `.vscode` to prevent further commits.
- make sure you've squashed your commits. `feat: Transfer to Pots` looks like a good one to use! In general it's keep it one commit per bug/feature/refactor etc...

Thanks again...good effort 👍

rossmuego commented:
> Good work mate! Pretty busy at the moment and then I'm on holiday from Friday, but I'll try and review properly asap.
> 
> But just a few things after a quick look:
> 
> * can you add `.DS_Store` to `.gitignore` so those files don't get committed?
> * although it looks like you've updated the PR to remove the files, we should probably add `.vscode` to prevent further commits.
> * make sure you've squashed your commits. `feat: Transfer to Pots` looks like a good one to use! In general it's keep it one commit per bug/feature/refactor etc...
> 
> Thanks again...good effort 👍

Addressed the .DS_Store and .vscode. Squashed all my commits however everytime I push after squashing I'm getting conflicts that I'm not getting in VSCode. Will be sure to squash when merging however 👍

johneas10 commented:
Hi @rossmuego 

Just back from holiday so I'll get a proper look this week.

In regards to your commit issues...It looks like you branched off a while back and didn't pull down the latest changes before adding your work. It's going to be a bit of a pain but you'll have to pull the latest from master and rebase that onto your PR/branch. You're probably going to run into a bunch or merge conflicts on the way but it's the only way to get your branch back in sync.

Other option is to pull the latest from master, create a new branch and add your work to that.

Cheers,
John
